### ISSUE TITLE
[Feature req] Add some command aliases

### ISSUE THREAD

**Issue by shrx** [8cylinder/tv-overlord#56]
As of now, the command names are a bit confusing:

```
[...]
  add          Add a new tv show to the database.
[...]
  editshow     Edit the contents of the database.
[...]
```
I suggest adding alias `addshow` to `add` command and `edit` to `editshow` command to reduce the confusion.

**Comment by 8cylinder** [comment]
I agree, the command names are not as intuitive as they could be.  That example you use is one I don't like myself and I also don't like `nondbshow`.

The framework I'm using ([Click](http://click.pocoo.org/6/)) doesn't support aliases.  It does support (what it calls aliases) being able to use part of a command: `tvol ca` instead of `tvol calendar`, but that's not what we're talking about here.

What I'll do is remove `show` from the commands, so `editshow` and `nondbshow` become `edit` and `nondb`.  The show is redundant and this also shortens the commands a bit for easier typing.

```
| Old         | New         |
|-------------+-------------|
| add         | add         |
| calendar    | calendar    |
| config      | config      |
| download    | download    |
| editshow    | edit        |
| history     | history     |
| info        | info        |
| list        | list        |
| nondbshow   | nondb       |
| re-copy     | re-copy     |
| re-download | re-download |
| re-search   | re-search   |
```
